modules/pubmaticBidAdapter.md

```diff
@@ -26,7 +26,7 @@ var adUnits = [
       params: {
         publisherId: '156209',               // required, must be wrapped in quotes
         oustreamAU: 'renderer_test_pubmatic',   // required if mediaTypes-> video-> context is 'outstream' and optional if renderer is defined in adUnits or in mediaType video. This value can be get by BlueBillyWig Team.
-        adSlot: 'pubmatic_test2',            // optional
+        adSlot: 'pubmatic_test2',            // optional, must be wrapped in quotes
```

Comment:
must be wrapped in quotes is an odd way of saying must be a string, when i first read it, i thought it meant must have escaped quotes as the first and last character of the string. 

how about: must be a string, not an integer or other js type.

Reply:
I copied the text that pubmatic (i assume) used on publisherId.  I'll update both to say String, which i agree makes more sense.